README.md

```diff
@@ -7,125 +11,131 @@ Skuber is a Scala client library for [Kubernetes](http://kubernetes.io). It prov
 
 See the [programming guide](docs/GUIDE.md) for more details.
 
-## Example Usage
-
-This example creates a nginx service (accessed via port 30001 on each Kubernetes cluster node) that is backed by a deployment of five nginx replicas.
-
-```scala
-import skuber._
-import skuber.json.format._
-import skuber.apps.v1.Deployment
-import LabelSelector.dsl._
-
-val nginxSelector  = "app" is "nginx" 
-val nginxContainer = Container(name = "nginx", image = "nginx").exposePort(80) 
-val nginxTemplate = Pod.Template.Spec.named("nginx").addContainer(nginxContainer).addLabel("app" -> "nginx")
-val nginxDeployment = Deployment(name)
-    .withReplicas(5)
-    .withTemplate(nginxTemplate)
-    .withLabelSelector(nginxSelector)
-val nginxService = Service("nginx")
-    .withSelector("app" -> "nginx")
-    .exposeOnNodePort(30001 -> 80) 
-
-// Some standard Akka implicits that are required by the skuber v2 client API
-import akka.actor.ActorSystem
-import akka.stream.ActorMaterializer
-implicit val system = ActorSystem()
-implicit val materializer = ActorMaterializer()
-implicit val dispatcher = system.dispatcher
-
-// Initialise skuber client
-val k8s = k8sInit
-
-// Create the service and the deployment on the Kubernetes cluster
-val createOnK8s = for {
-  svc <- k8s create nginxService
-  dep  <- k8s create nginxDeployment
-} yield (dep,svc)
-
-createOnK8s onComplete {
-  case Success(_) => System.out.println("Successfully created nginx deployment & service on Kubernetes cluster")
-  case Failure(ex) => System.err.println("Encountered exception trying to create resources on Kubernetes cluster: " + ex)
-}
-
-k8s.close // this prevents any more requests being sent by the client
-system.terminate // this closes the connection resources etc.
-```
+## Quick Start
 
-## Prerequisites
+Make sure [prerequisites](#prerequisites) are met. There are couple of quick ways to get started with Skuber:
 
-A Kubernetes cluster is needed at runtime. For local development purposes, minikube is recommended.
+### With [Ammonite-REPL](http://ammonite.io/#Ammonite-REPL) 
 
-You need Java 8 to run Skuber.
+Provides you with a configured client on startup. It is handy to use this for quick experiments. 
+- using bash
+    
+  ```bash
+  $ amm -p ./Quickstart.sc
+  ```
 
-## Release
+- from inside ammonite-repl:
+  ```scala
+  import $file.`Quickstart`, Quickstart._
+  ```
 
-You can use the latest release (for Scala 2.11 or 2.12) by adding to your build:
+  ###### Just handy shortcut to import skuber inside ammonite-repl:
 
-```sbt
-libraryDependencies += "io.skuber" %% "skuber" % "2.0.9"    
-```
+  ```
+  import $ivy.`io.skuber::skuber:2.0.9`, skuber._, skuber.json.format._
+  ```
 
-Meanwhile users of skuber v1 can continue to use the latest (and possibly final, with exception of important fixes) v1.x release, which is available only on Scala 2.11:
+### Copy-paste ready snippet
```

Comment:
As a sub-heading can we retitle this to `Example`, and note that t can be copied/pasted in the first line of the following paragraph?

Reply:
Also I'd prefer this to be in its own section before the quickstart, placing it between the ammonite and sbt parts of the quickstart could be a bit confusing,

Reply:
ack